docs/PfSense.md

```diff
@@ -0,0 +1,35 @@
+In order to boot PfSense via PXE,
+you need to do the following:
+
+1. Download ISO from https://nyifiles.pfsense.org/mirror/downloads/pfSense-CE-2.3.4-RELEASE-amd64.iso.gz
+   
+   You should also check the SHA256 hash:
+   `sha256 -c 610b2a8c696e1d1854845d553ce8540debaab9d78ed6c15ce83872eaeac9d05f pfSense-CE-2.3.4-RELEASE-amd64.iso.gz`
+   and compare it to the one in https://nyifiles.pfsense.org/mirror/downloads/pfSense-CE-2.3.4-RELEASE-amd64.iso.gz.sha256
+
+2. Extract the ISO contents to a directory called
+    `images/pfsense` in TFTP root:
+    `gzip -d pfSense-CE-2.3.4-RELEASE-amd64.iso.gz`
+    
+   In order to extract the contents, you need to mount
+   the ISO with following commands (example from FreeBSD):
+   `mount_cd9660 /dev/$(mdconfig -a -t vnode -f pfSense-CE-2.3.4-RELEASE-amd64.iso) /mnt`
+   Then:
+   `rsync -avvP /mnt/ /srv/tftp/images/pfsense/`
+   You need to have `rsync` installed.
+
+2. Relevant entry in DHCP server config:
+    `option root-path /srv/tftp/images/pfsense;`
```

Comment:
@pkubaj I would appreciate to prepare a solution for a network without running DHCP server on linux machine. Please provide necessary kernel command line with nfs options if possible. I will be able then to reproduce it

Reply:
Sorry, this time you must adjust to the standard :)
https://www.freebsd.org/doc/handbook/network-diskless.html
"In order to provide the files needed for an operating system to boot over the network, a PXE setup also requires properly configured DHCP, TFTP, and NFS servers"